client/ayon_hiero/api/plugin.py

```diff
@@ -711,30 +745,23 @@ def convert(self):
         if self.rename:
             # rename track item
             self.track_item.setName(new_name)
-            self.tag_data["asset_name"] = new_name
+            self.tag_data["asset"] = new_name
```

Comment:
AYON doesn't have "assets" - instead it has "folders"
This should maybe be `folder_name` if it's not the full folder path, otherwise `folder_path`?

Reply:
Asset should be really `folderName`. But we should make sure backward compatibility for cases where tags were created before the update.